dask_ml/model_selection/_incremental.py

```diff
@@ -757,7 +859,58 @@ def _get_params(self):
         else:
             return ParameterSampler(self.parameters, self.n_initial_parameters)
 
+    def _stop_on_plateau(self, info, instructions):
+        _best_plateau = {}
```

Comment:
Also, style nit: this is a function, so you don't need to prefix variable names with `_` to indicate privacy.